docs/api/web-contents.md

```diff
@@ -2064,6 +2064,21 @@ Setting the WebRTC IP handling policy allows you to control which IPs are
 exposed via WebRTC. See [BrowserLeaks](https://browserleaks.com/webrtc) for
 more details.
 
+#### `contents.getWebRTCUDPPortRangePolicy()`
+
+Returns `Object`:
+
+* `min` Integer - The minimum port number of WebRTC UDP Port Range Policy.
+* `max` Integer - The maximum port number of WebRTC UDP Port Range Policy.
+
+#### `contents.setWebRTCUDPPortRangePolicy(udpPortRange)`
+
+* `udpPortRange` Object
+  * `min` Integer - The minimum port number of WebRTC UDP Port Range Policy.
+  * `max` Integer - The maximum port number of WebRTC UDP Port Range Policy.
+
+Setting the WebRTC UDP Port Range policy allows you to restrict the udp port range assigned to apps via WebRTC.
```

Comment:
If someone sets a policy and then wants to remove it, would they pass `{ min: 0, max: 0 }`? Is it worth including that in the docs?

Reply:
Sure, this is a reasonable scene, thanks for your review.